client/src/containers/frontend/Featured.js

```diff
@@ -43,6 +43,13 @@ export class FeaturedContainer extends Component {
     subjects: PropTypes.array,
     dispatch: PropTypes.func
   };
+
```

Comment:
This is necessary if you're on the /featured page and log in.  You won't see the draft projects unless we refetch.